README.md

```diff
@@ -1,2 +1,168 @@
+
+### /users/$userId$/messages/$messageId$
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|dueDate|optional DateTime|-|The due date of the message to be shown in-app.|
+|triggerDate|optional DateTime|-|A date to trigger a notification on the device. If not set, then there is no notification to trigger. The notification should be triggered by the server. TBD: Are there cases where a message would trigger multiple times or event- rather than date-driven?|
+|completionDate|optional DateTime|-|Specifies when a message has been completed. TBD: Messages containing a completionDate may either be hidden on user's devices or be shown crossed out.|
+|title|LocalizedText|e.g. "Watch Welcome Video in Education Page."|May be localized.|
+|description|optional LocalizedText|e.g. "The video shows how you will be able to use this app."|May be localized.|
+|action|optional string|e.g. "engage-hf:/videoSections/engage-hf/videos/welcome"|Format to be defined, needs to be decoded on a device and performed on tap of the message.|
+
+### /users/$userId$/allergyIntolerances/$allergyIntoleranceId$
+
+Based on [FHIR AllergyIntolerance](https://hl7.org/fhir/R4B/allergyintolerance.html), the following properties may be used, while additional properties are ignored by the Engage-HF system.
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|-|[Resource](https://hl7.org/fhir/R4B/resource.html): Logical id of this artifact|
+|type|optional [allergyIntoleranceType](https://hl7.org/fhir/R4B/valueset-allergy-intolerance-type.html)|e.g. "allergy", "intolerance"||
+|code|CodableContent|e.g. "{"coding":[{"system":"https://hl7.org/fhir/R4B/valueset-allergyintolerance-code.html","code":"293963004","display":"Cardioselective beta-blocker allergy"}],"text":"Cardioselective beta-blocker allergy"}"|Uses either [AllergyIntoleranceCode](https://hl7.org/fhir/R4B/valueset-allergyintolerance-code.html), `medicationId` as used in /medications/$medicationId$ and/or `medicationId` as used in /medicationClasses/$medicationClassId$.|
+|patient|string|e.g. "allergy", "intolerance"|`userId` as used in /users/$userId$ and related collections.|
+
+### /users/$userId$/medicationRequests/$medicationRequestId$
+
+Based on [FHIR MedicationRequest](https://hl7.org/fhir/R4B/medicationrequest.html), the following properties may be used, while additional properties are ignored by the Engage-HF system.
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|-|[Resource](https://hl7.org/fhir/R4B/resource.html): Logical id of this artifact|
+|type|optional [allergyIntoleranceType](https://hl7.org/fhir/R4B/valueset-allergy-intolerance-type.html)|e.g. "allergy", "intolerance"||
```

Comment:
I think there are some copy and paste errors here.

Reply:
✅  should be fixed now